raiden-ts/src/raiden.ts

```diff
@@ -303,6 +303,8 @@ export class Raiden {
         error: 'error',
         nextState: 'debug',
       },
+      // disable colors outside of browsers env
+      ...(globalThis?.location ? {} : { colors: false }),
```

Comment:
I think the quality can be improved by removing the comment and express it as code. So just have something like `const beingInsideBrowser = !!globalThis?.location` and then use it directly in the object assignment. Results into the exact same number of lines but with more value. :grimacing:

Reply:
:heavy_check_mark: